src/library/mod.rs

```diff
@@ -28,28 +71,16 @@ impl ftd::p2::Library for Library {
                 ),
                 _ => fpm_base,
             })
-        } else if let Ok(v) = std::fs::read_to_string(format!("./{}.ftd", name)) {
-            Some(v)
-        } else if let Ok(v) = std::fs::read_to_string(format!("./.packages/{}.ftd", name)) {
-            Some(v)
         } else {
-            std::fs::read_to_string(format!("./.packages/{}/index.ftd", name)).ok()
+            std::fs::read_to_string(format!("./{}.ftd", name)).ok()
         }
     }
 
     fn process(
         &self,
-        section: &ftd::p1::Section,
-        doc: &ftd::p2::TDoc,
+        _section: &ftd::p1::Section,
+        _doc: &ftd::p2::TDoc,
     ) -> ftd::p1::Result<ftd::Value> {
-        match section
-            .header
-            .str(doc.name, section.line_number, "$processor$")?
-        {
-            "toc" => fpm::library::toc::processor(section, doc),
-            "http" => fpm::library::http::processor(section, doc),
-            "package-query" => fpm::library::sqlite::processor(section, doc),
-            t => unimplemented!("$processor$: {} is not implemented yet", t),
-        }
+        unimplemented!("not implemented yet");
```

Comment:
You can just delete this entire function. We have a default implementation in `ftd::Library`.